packages/app/src/components/add-tag/AddTagStepTrack.vue

```diff
@@ -109,7 +109,7 @@ export default defineComponent({
 
       const tagCopy = tag.value;
       tagCopy.trackUri = track.uri;
-      tagCopy.imageUrl = track.imageUri;
+      tagCopy.trackImageUrl = track.imageUri;
```

Comment:
What is the diffrence between `tagCopy.trackImageUrl` and `  tagCopy.imageUrl`
The naming is not quite clear

Is imageUri the external image?

Reply:
trackImageUrl is the default image from the music service(i. e. spotify). trackImage is the image Url th user chooses

Reply:
Could we rename that to clarify that?
- externalImageUri
- musicServiceImageUri

or something like this?

Reply:
@JanEberlein I would simplify that a bit as well. I will work on it and commit my changes.

Reply:
I resolved that ...